sos/report/plugins/greenboot.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2023 Red Hat, Inc. Evgeny Slutsky <eslutsky@redhat.com>
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+
+
+class Greenboot(Plugin, RedHatPlugin):
+    """The greenboot plugin collects systemd service logs and configuration.
+    """
+
+    short_desc = 'Greenboot'
+    plugin_name = 'greenboot'
+    plugin_timeout = 300
+    packages = ('microshift-greenboot',)
```

Comment:
We probably don't want to tie this to MicroShift.

Reply:
@dhellmann , greenboot is distributed as `microshift-greenboot` rpm, are we going to change that also?

Reply:
The `microshift-greenboot` RPM contains the scripts for MicroShift to integrate with greenboot, but the actual system software is in the `greenboot` RPM.